statsmodels/datasets/tests/test_utils.py

```diff
@@ -18,8 +18,9 @@ def test_get_rdataset():
         duncan = get_rdataset("Duncan", "car", cache=cur_dir)
         assert_(duncan.from_cache)
 
-# temporarily disabled because some test servers don't have internet
-def t_est_webuse():
+internet_available = check_internet()
+@dec.skipif(internet_available is False)
```

Comment:
not for booleans.
we don't write `if  internet_available is True: ....` either, unless we explicitly want to rule out everything else that might evaluate to true.